.github/workflows/checkov.yml

```diff
@@ -4,9 +4,18 @@ on:
   push:
     branches:
       - main
+    paths:
```

Comment:
IMO we shouldn't be running any tests unnecessarily - if the changeset isn't being tested by a given test, we shouldn't run the test (because a failure at that point is just confusing).

The actual jobs in these tests are specifically scoped to the same paths I've listed here. ie. if someone were to add a new CFN template to the repository, they'd have to specifically update this file for that file to be tested anyway, so my assumption is that 'did the updated workflow run properly' would be part of the PR review. So if the `on` wasn't updated, it'd be noticed and corrected.

Thoughts?